octobot_trading/exchanges/config/ccxt_exchange_settings.pxd

```diff
@@ -9,15 +10,10 @@
 #  This library is distributed in the hope that it will be useful,
 #  but WITHOUT ANY WARRANTY; without even the implied warranty of
 #  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-#  Lesser General Public License for more details.
+#  Lesser General License for more details.
```

Comment:
?

Reply:
Dont know why I removed that, I added it again